orangecontrib/spectroscopy/data.py

```diff
@@ -700,6 +700,17 @@ def read(self):
                 else:
                     meta_data = params
 
+        visible_images = []
+        for img in opusFC.getVisImages(self.filename):
+            visible_images.append({
+                'name': img['Title'],
+                'image_bytes': img['image'],
+                'pos_x': img['Pos. X'] * img['PixelSizeX'],
+                'pos_y': img['Pos. Y'] * img['PixelSizeY'],
+                'pixel_size_x': img['PixelSizeX'],
+                'pixel_size_y': img['PixelSizeY'],
+            })
+
```

Comment:
@stuart-cls, is this safe? Do images always contain these attributes or should we wrap in try ... except?

Reply:
We should wrap. In opusFC I just directly copy the attributes from the file, so if Bruker has used different labels in the past/future, the name will change. Unfortunately, the visible images part is not documented so I had to do some guessing there.